### DISCUSSION TITLE
Grids in as in complex sample "Sidebar & grid" - lost selection of record

### DISCUSSION THREAD

**Discussion by Rimokas** [vitmalina/w2ui#2338] [Q&A]
Hi!
need some help, can't understand what is wrong.

Did page, very similar as in "Demos" -> Sidebar & grid ( https://w2ui.com/web/demos/#/combo/1 )
All is working, nice view. 
The difference is that mine grids data are from remote datasource (mySql), with urls and etc.
 
In sample if I click in grid1 3-rd record and then go to grid2, after returning to grid1 I will find the same selected record. And that is what I want.
With mine grids with remote data, after returning to grid1 
   `                case 'grid1':
                    layout.html('main', grid1)
                    break
`
data is reloading and I loose selected record. The same and with grid2 too ... :( 

How to let for grids do not reload data after rendering to layout panel ?

Thanks in advance for any help ... :)

**Comment by vitmalina** [comment]
Well, grid will try to reload data if it is rendered and if there is a `url` property. What you could do, you is to use `load` method, see https://w2ui.com/web/docs/2.0/w2grid.load. In this case it will only load when you ask it.
